components/brave_new_tab_ui/components/default/together/index.tsx

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import * as Crypto from 'crypto'
+import createWidget from '../widget/index'
+import { getLocale } from '../../../../common/locale'
+
+import {
+  WidgetWrapper,
+  Header,
+  Content,
+  WelcomeText,
+  ActionsWrapper,
+  CallButton,
+  TogetherIcon,
+  StyledTitle,
+  Privacy,
+  PrivacyLink
+} from './style'
+import { StyledTitleTab } from '../widgetTitleTab'
+import BraveTogetherIcon from './assets/brave-together-icon'
+
+interface Props {
+  showContent: boolean
+  onShowContent: () => void
+}
+
+class Together extends React.PureComponent<Props, {}> {
+
+  getButtonText = () => {
+    return getLocale('togetherWidgetStartButton')
+  }
+
+  renderTitle () {
+    return (
+      <Header>
+        <StyledTitle>
+          <TogetherIcon>
+            <BraveTogetherIcon />
+          </TogetherIcon>
+          <>
+            {getLocale('togetherWidgetTitle')}
+          </>
+        </StyledTitle>
+      </Header>
+    )
+  }
+
+  renderTitleTab () {
+    const { onShowContent } = this.props
+
+    return (
+      <StyledTitleTab onClick={onShowContent}>
+        {this.renderTitle()}
+      </StyledTitleTab>
+    )
+  }
+
+  shouldCreateCall = (event: any) => {
+    event.preventDefault()
+
+    Crypto.randomBytes(32, (err, buffer) => {
```

Comment:
@fmarier has said using the normal web crypto api is ok here, so let's replace.

@diracdeltas pointed to one that looks like it boils down to just the following, which could go in to a `getRandomBytes.ts` file:

```js
const QUOTA = 65536
export default function getRandomBytes(n) {
            var a = new Uint8Array(n)
            for (var i = 0; i < n; i += QUOTA) {
              crypto.getRandomValues(a.subarray(i, i + Math.min(n - i, QUOTA)))
            }
            return a
          }
```

Reply:
This would be used like this:
```js
btoa(String.fromCharCode.apply(null, getRandomBytes(32))).replace(/\+/g, '-').replace(/\//g, '_').replace(/\=/g, '')
// -> UgfDL4qJTVRl3EaTTrTGi6BFBV8UnLvmbsQC8u8S_Uw
```

Reply:
> ```js
> const QUOTA = 65536
> export default function getRandomBytes(n) {
>   var a = new Uint8Array(n)
>   for (var i = 0; i < n; i += QUOTA) {
>     crypto.getRandomValues(a.subarray(i, i + Math.min(n - i, QUOTA)))
>   }
>   return a
> }

If we alway use 32 bytes, isn't that quota check unnecessary?

Doesn't that just end up being the following?
```
a = new Uint8Array(32)
crypto.getRandomValues(a)
```

Reply:
i think we won't really need more than 32 bytes so it's fine to simplify it the way @fmarier suggested. would be nice to have a test for this that checks the output against the check on the server: https://github.com/brave/jitsi-meet/commit/b897e8a0e065f5ac93b88fcbc3b48d3d56a7b45c

Reply:
this has been addressed